.github/workflows/dt.yaml

```diff
@@ -25,15 +25,26 @@ jobs:
           git config user.password ${{ secrets.DEFINITELY_TYPED_PR_GITHUB_ACCESS_TOKEN }}
           git remote set-url origin https://github.com/dbrudner/DefinitelyTyped
           git fetch --unshallow -p origin
+      - name: Add DefinitelyTyped header
+        run: |
+          version=$(cat recurly-js/package.json | jq '.version' -r | sed -ne 's/^\([0-9]*\.[0-9]*\).*/\1/p')
+          echo "// Type definitions for non-npm package recurly__recurly-js $version
+          // Project: https://github.com/recurly/recurly-js
+          // Definitions by: Dave Brudner <https://github.com/dbrudner>
+          //                 Chris Rogers <https://github.com/chrissrogers>
+          // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+          // TypeScript Version: 3.1" > index.d.ts
+          cat recurly-js/types/index.d.ts >> index.d.ts
+          mv index.d.ts recurly-js/types
```

Comment:
echo "// Type definitions for non-npm package recurly__recurly-js $version
// Project: https://github.com/recurly/recurly-js
// Definitions by: Dave Brudner <https://github.com/dbrudner>
//                 Chris Rogers <https://github.com/chrissrogers>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
// TypeScript Version: 3.1
$(cat recurly-js/types/index.d.ts)" > recurly-js/types/index.d.ts

Reply:
Updated https://github.com/recurly/recurly-js/blob/014c2a65c83b30bd6d351972ab91a68dc16f6b0c/.github/workflows/dt.yaml#L28-L37